contracts/batcher/Batcher.sol

```diff
@@ -28,31 +36,61 @@ abstract contract Batcher is IBatcher, UOwnable {
         __UOwnable__initialize();
     }
 
+    /**
+     * @notice Total USDC and DSU balance of the Batcher
+     * @return Balance of DSU + balance of USDC
+     */
     function totalBalance() public view returns (UFixed18) {
```

Comment:
qq: did we decide on keeping this?

Reply:
Yeah we use it in the `rebalance` flow and in the rebalancer defender task so I opted to just keep it for now